module.yml

```diff
@@ -442,15 +442,13 @@ Resources:
                 runas=root
           services:
             sysvinit:
+              # amazon-ssm-agent is started by upstart
               cfn-hup:
                 enabled: true
                 ensureRunning: true
                 files:
                 - '/etc/cfn/cfn-hup.conf'
                 - '/etc/cfn/hooks.d/cfn-auto-reloader.conf'
-              amazon-ssm-agent:
```

Comment:
the problem is that ssm is not stopped now if you turn it off.
I recognized a 3-5 min delay because of some issue with ssm to be started twice. Was this the same problem you experienced?

Reply:
It is not possible to manage the service `amazon-ssm-agent` with `sysvinit`. Instead `amazon-ssm-agent` uses `upstart` on Amazon Linux (see https://docs.aws.amazon.com/systems-manager/latest/userguide/sysman-manual-agent-install.html). Therefore, it is not possible to configure the service with cfn-init. The instance failed to launch.

You experienced an issue when terminating an instance because the SSM agent was causing a 3-5 minutes delay?